SPECS/systemd/systemd.spec

```diff
@@ -1,62 +1,342 @@
-%define majver %(echo %{version} | cut -d. -f 1)
-Summary:        Systemd
+
+# Temporary until azure-release-common is updated to carry macros.dist that defines %azure
+%global azure 3
+
+# We ship a .pc file but don't want to have a dep on pkg-config. We
+# strip the automatically generated dep here and instead co-own the
+# directory.
+%global __requires_exclude pkg-config
+
+%global pkgdir %{_prefix}/lib/systemd
+%global system_unit_dir %{pkgdir}/system
+%global user_unit_dir %{pkgdir}/user
+
+%if 0%{?__isa_bits} == 64
+%global elf_bits (64bit)
+%global elf_suffix ()%{elf_bits}
+%endif
+
+%bcond bzip2  1
+%bcond gnutls 0
+%bcond lz4    1
+%bcond xz     1
+%bcond zlib   1
+%bcond zstd   1
+
+# Bootstrap may be needed to break circular dependencies with cryptsetup,
+# e.g. when re-building cryptsetup on a json-c SONAME-bump.
+%bcond bootstrap 0
+%bcond tests     1
+%bcond lto       1
+
+# Support for quick builds with rpmbuild --build-in-place.
+# See README.build-in-place.
+%bcond inplace 0
+
 Name:           systemd
-Version:        254.5
-Release:        1%{?dist}
-License:        LGPLv2+ AND GPLv2+ AND MIT
-Vendor:         Microsoft Corporation
-Distribution:   Mariner
-Group:          System Environment/Security
-URL:            https://www.freedesktop.org/wiki/Software/systemd/
-Source0:        https://github.com/%{name}/%{name}-stable/archive/refs/tags/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
-Source1:        95-disable-systemd-oomd.preset
-BuildRequires:  audit-devel
-BuildRequires:  cryptsetup-devel
-BuildRequires:  docbook-dtd-xml
-BuildRequires:  docbook-style-xsl
-BuildRequires:  gettext
-BuildRequires:  glib-devel
-BuildRequires:  gperf
-BuildRequires:  intltool
-BuildRequires:  kbd
-BuildRequires:  kmod-devel
+Url:            https://systemd.io
+%if %{without inplace}
+Version:        255
```

Comment:
This is an enormous change. Was there an issue with a simple version bump and getting the new sources? If this change is about more than just updating the version of `systemd`, please also include these details in the PR's summary.

Reply:
yes, we need to shift to fedora's packaging

Reply:
I'm not sure this is our current goal or approach right now. Having said that, I believe you mentioned that the subpackage splitting you've done here allowed to reduce the size of some of our images, so the change is definitely valuable. Please only mention that in the PR's summary to make it easier in the future to understand the reasoning which lead us to this update.